src/explore-education-statistics-admin/src/pages/release/api-data-sets/ReleaseApiDataSetDetailsPage.tsx

```diff
@@ -0,0 +1,195 @@
+import Link from '@admin/components/Link';
+import { useConfig } from '@admin/contexts/ConfigContext';
+import ApiDataSetVersionSummaryList from '@admin/pages/release/api-data-sets/components/ApiDataSetVersionSummaryList';
+import { useReleaseContext } from '@admin/pages/release/contexts/ReleaseContext';
+import apiDataSetQueries from '@admin/queries/apiDataSetQueries';
+import {
+  releaseApiDataSetsRoute,
+  ReleaseDataSetRouteParams,
+  ReleaseRouteParams,
+} from '@admin/routes/releaseRoutes';
+import ButtonText from '@common/components/ButtonText';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import SummaryCard from '@common/components/SummaryCard';
+import Tag from '@common/components/Tag';
+import TaskList from '@common/components/TaskList';
+import TaskListItem from '@common/components/TaskListItem';
+import { useQuery } from '@tanstack/react-query';
+import React from 'react';
+import { generatePath, useParams } from 'react-router-dom';
+
+// TODO: EES-4370
+const showRemoveDraft = false;
+// TODO: Version mapping
+const showDraftVersionTasks = false;
+
+// TODO: EES-4367
+const showChangelog = false;
+// TODO: ES-4382
+const showVersionHistory = false;
+
+export default function ReleaseApiDataSetDetailsPage() {
```

Comment:
As mentioned on Slack, I get a 500 for `https://ees.local:5021/api/public-data/data-sets/<id>` when I try to view data set details after a new release has been published (it works ok for the seed data).